src/utils/dlo.js

```diff
@@ -0,0 +1,29 @@
+import {insertScriptIntoDocument} from "./browser";
```

Comment:
This file was pulled out so registerRules could be called directly and so I can test the path where dlo has already been loaded.  I could not figure out an easy way to accomplish both of these without a separate import.  I could export the function from the main script, but that left an ugly export in the IIFE.